app/uk/gov/hmrc/tradergoodsprofiles/controllers/actions/AuthAction.scala

```diff
@@ -117,8 +117,8 @@ class AuthActionImpl @Inject() (
 
     Future.successful(
       ForbiddenErrorResponse(
-        dateTimeService.timestamp,
-        s"This EORI number is incorrect"
+        uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet